src/test/kotlin/org/rust/ide/annotator/RsAttrErrorAnnotatorTest.kt

```diff
@@ -144,4 +160,82 @@ class RsAttrErrorAnnotatorTest : RsAnnotatorTestBase(RsAttrErrorAnnotator::class
         #[cfg(feature = "foo")]
         fn bar() {}
     """)
+
+    fun `test duplicates remove first`() = checkFixByText("Remove `#[instruction_set(arm::a32)]`", """
+        #[instruction_set(arm::a32)]
+        #[<error descr="Multiple 'instruction_set' attributes">instruction_set(arm::a32)/*caret*/</error>]
+        fn foo_arm_code() {}
+    """, """
+        #[instruction_set(arm::a32)]
+        fn foo_arm_code() {}
+    """)
+
+    fun `test duplicates remove last`() = checkFixByText("Remove `#[ignore]`", """
+        #[<warning descr="Unused attribute">ignore/*caret*/</warning>]
+        #[ignore]
+        fn foo() {}
+    """, """
+        #[ignore]
+        fn foo() {}
+    """)
+
+    fun `test deprecated attr E0538`() = checkByText("""
+        #[deprecated]
+        fn foo() {}
+
+        #[deprecated = "hello world"]
+        fn bar() {}
+
+        #[deprecated(
+            since = "a",
+            <error descr="Multiple 'since' items [E0538]">since = "b"</error>,
+            note = "c"
+        )]
+        fn f1() { }
+
+        #[deprecated(
+            since="1.0.0",
+            note="First deprecation note.",
+            <error descr="Multiple 'note' items [E0538]">note="Second deprecation note."</error>
+        )]
+        fn f2() {}
+    """)
+
+    fun `test remove multiple item E0538`() = checkFixByText("Remove `since = \"b\"`", """
+        #[deprecated(
+            since = "a",
+            <error descr="Multiple 'since' items [E0538]">since = "b"/*caret*/</error>,
+            note = "c"
+        )]
+        fn f1() { }
+    """, """
+        #[deprecated(
+            since = "a",
+        ,
```

Comment:
The quick fix should also remove the comma. I think it's better just remove the quick-fix, merge the PR without it, and maybe make a separate PR with the adjusted quick-fix